src/main/java/com/redhat/cloud/common/clowder/configsource/ClowderConfigSourceFactory.java

```diff
@@ -27,14 +27,19 @@ public class ClowderConfigSourceFactory implements ConfigSourceFactory {
     public Iterable<ConfigSource> getConfigSources(ConfigSourceContext configSourceContext) {
 
         ConfigValue cv = configSourceContext.getValue("acg.config");
-        String clowderConfig;
+        ConfigValue exposeKafkaSslConfigKeysCv = configSourceContext.getValue("acg.config.expose.kafka.ssl.config.keys");
```

Comment:
`acg.config` is something that comes from Clowder, we don't need it it this feature flag key.

We could use a slightly different key here and make it obvious that it is a feature flag:

```suggestion
        ConfigValue exposeKafkaSslConfigKeysCv = configSourceContext.getValue("feature-flags.expose-kafka-ssl-config-keys.enabled");
```